beamer/tests/agent/test_request.py

```diff
@@ -302,3 +302,23 @@ def test_agent_ignores_invalid_fill(_, request_manager, token, agent: Agent):
     with Sleeper(1) as sleeper:
         while not request.is_filled:
             sleeper.sleep(0.1)
+
+
+def test_unsafe_fill_time(
+    request_manager, config, token, set_allow_unlisted_pairs
+):  # pylint:disable=unused-argument
+    requester, target = alloc_accounts(2)
+    request_id = make_request(request_manager, token, requester, target, 1)
+    # 1 second to fill the request
+    config.unsafe_fill_time = 1799
```

Comment:
Nitpick: might be better to compute the value based on the contract's max validity time.

Reply:
then agent just raise runtime error ?

Reply:
> then agent just raise runtime error ?

I'm not sure what you mean, but I meant something like
```
config.unsafe_fill_time = RequestManager.MAX_VALIDITY_PERIOD - 1
```

Reply:
ok now got it :)